src/components/AddItems.js

```diff
@@ -36,6 +42,24 @@ const AddItemsToList = () => {
       .get()
       .then((data) => {
         if (data.docs.length) {
+          const { items } = data.docs[0].data();
+          const nonWordCharactersAndUnderscores = /[\W_]/g;
```

Comment:
I have two suggestions for `nonWordCharactersAndUnderscores`
1. Since it's a constant string it would better positioned at the beginning of the file with the other constants (`db`, `arrayUnion`)
2. Or you can put it in the normalize function I suggest below